site/en/docs/webstore/faq/index.md

```diff
@@ -65,6 +66,10 @@ now pay it in order to continue accessing your developer dashboard. This one-tim
 the standard [developer registration process][99].
 {% endAside %}
 
+### I'm having trouble registering my developer account {: #faq-gen-33 }
+
+If you encounter any problems during the developer registration process, try using an incognito window to register. 
```

Comment:
Is "any problems" too broad? Like: can't remember username, no internet connectivity, etc. aren't helped by incognito. Is there a concise way to express the class of problems that this could address?

Reply:
I opted to remove this for now. I will make a note to do further research to identify specific problems.